nixos/modules/services/networking/soju.nix

```diff
@@ -25,6 +25,10 @@ let
 
     ${cfg.extraConfig}
   '';
+
+  sojuctl = pkgs.writeShellScriptBin "sojuctl" ''
+    exec ${pkgs.soju}/bin/sojuctl --config ${configFile} "$@"
+  '';
```

Comment:
We should use wrapProgram for this. Also it might be worth it to add a package option, too.

Reply:
Making it a package option is a little weird, since it depends on the service module's `configFile` derivation. I don't know of any other packages that work like that.

For `wrapProgram`, are you suggesting `mkDerivation` in the service module that calls `wrapProgram` in `installPhase`, or something else?

Reply:
> Making it a package option is a little weird, since it depends on the service module's `configFile` derivation.

I mean replacing ``pkgs.soju`` with ``cfg.package``.

> For wrapProgram, are you suggesting mkDerivation in the service module that calls wrapProgram in installPhase, or something else?

The longer I think about it, the more I think we can just leave it as is.

Reply:
> I mean replacing pkgs.soju with cfg.package.

Ah, that's a good idea, added.